extensions/wikia/DesignSystem/DesignSystemHelper.class.php

```diff
@@ -108,7 +108,7 @@ private static function getCachedSvg( $name ) {
 	 *
 	 * @return string
 	 */
-	public static function renderText( $fields, $recursionDepth = 0 ) {
+	public static function renderText( $fields, $inContentLang = false, $recursionDepth = 0 ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this function 5, down to the maximum allowed 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)